bin/grade_items_loop.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+from datetime import datetime
+import os
+import portalocker
```

Comment:
If we're not targeting Windows, let's just use the builtin python library [fcntl](https://docs.python.org/3/library/fcntl.html)?